<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projekt JPO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="updateMainDatabase">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="ListOfStations">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>256</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>City</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="ListOfSensor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Sensor Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sensor Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="areaForChart" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
